roles/ceph-nfs/tasks/pre_requisite.yml

```diff
@@ -1,30 +1,33 @@
 ---
-- name: create nfs gateway directories
-  file:
-    path: "{{ item }}"
-    state: directory
-    owner: "ceph"
-    group: "ceph"
-    mode: "0755"
-  with_items:
-    - /var/lib/nfs/ganesha
-    - /var/run/ganesha
-
 - name: create rgw nfs user
   command: radosgw-admin --cluster {{ cluster }} user create --uid={{ ceph_nfs_rgw_user }} --display-name="RGW NFS User"
   register: rgwuser
+  delegate_to: "{{ groups[rgw_group_name][0] }}"
```

Comment:
@leseb when I tried delegating this task to mon nodes I had issues, but doing them on the rgw seemed to go just fine.

Reply:
Hum, then rgw nodes must have the admin key.
What was the problem? This shouldn't different from a mon or an osd